### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN011.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#57] [General]
# https://blog.3place.cn/3place/lmyn/LMYN011.html

https://blog.3place.cn/3place/lmyn/LMYN011.html

**Comment by picklecai** [comment]
他将同鸯伽国王
结成亲密的友情，
他〔142〕将有超群的女儿，
散他就是她的名。 （1.10.3）

这个鸯伽国王的儿子，
把卢摩婆陀这名字起。

就是这一段，看着特别乱。
散他是鸯伽王的女儿，卢摩婆陀是鸯伽王自己的名字。
我看了这一段，还以为冒出来一个鸯伽王的儿子卢摩婆陀。
